### ISSUE TITLE
only show connecting 

### ISSUE THREAD

**Issue by monikarora** [GleasonK/AndroidRTC#22]
Hey @GleasonK  After accepting incoming call , only  showing connecting .not connected to other devices

**Comment by mkmlh** [comment]
https://github.com/newbie007fx/newwebrtc
Try this one, itu because STUN or TURN Protocol not implemented. U have to try with the same Network for now.

**Comment by monikarora** [comment]
Hey @imkamalh  , i already used this but its not working

**Comment by mkmlh** [comment]
Is it with the same network?

**Comment by monikarora** [comment]
call button is not working

**Comment by mkmlh** [comment]
Can u send the line of codes? The thing is my project works fine with this https://github.com/newbie007fx/newwebrtc. The only I change the setup is pubnub account of mine , testing the demo with the same Network.

**Comment by monikarora** [comment]
Hey @imkamalh  please tell me  your app provide group video calling in ?my own credential  PUB_KEY & SUB_KEY are not working why ?
Thanks!

**Comment by mkmlh** [comment]
Yup, I clone the project from him (the one that I share the link), and it works like a charm. Try the pub and sub key from GleasonK demo WebRTC project, if it still not working maybe that's not the problem.

**Comment by monikarora** [comment]
Okay @imkamalh  now its working .thanks!
i'm using GleasonK demo pub & sub key 
please send me Group video calling link (i also want group video like conference video call )

**Comment by mkmlh** [comment]
Yup I'm glad to help @monikarora 
Lol, now that's the hard one. In conference video call we have to implements the different method ( technically the handshake & communication method between the peers). If someone know how to implements, we really really appreciate that 😀

**Comment by sukeshikontel** [comment]
How do we connect when the phones are on separate n/w @imkamalh

**Comment by mkmlh** [comment]
@sukeshikontel we must apply TURN server. And the bad news is we must pay some service to apply turn server.

**Comment by sukeshikontel** [comment]
thanks for the reply @imkamalh

**Comment by HassaanNasir2018** [comment]
App is not connecting one Android Phone to another Android Phone...Its just keep showing the Connecting text thats it...... Please I need to video char from one mobile to another not to the Web browser.

**Comment by HassaanNasir2018** [comment]
Thanks its working from one app to other app but when both mobile connects with different networks it only allows chatting and no video.. Why?

**Comment by phamquocdai123** [comment]
Hi, i run completely project android from https://github.com/newbie007fx/newwebrtc , i enter number but button Call not act anything, what is this number ?,anyone can help me please ? i have to install something else ?
